### ISSUE TITLE
Locator assertions don't auto-wait

### ISSUE THREAD

**Issue by abr-storm** [YusukeIwaki/playwright-ruby-client#268]
Hi, thanks for making this library

### Usecase / What to achieve
I'm trying to reproduce the behaviour of this JS snippet in ruby:
```js
await expect(page.getByRole('heading', { name: 'Not on the page'})).toBeVisible()
```
It will trigger Playwright's auto-wait and raise a timeout error if it's not found after the time.

### Question
I would expect this ruby code to behave in the same way, but it doesn't. It fails immediately without waiting:
```ruby
expect(@page.get_by_role('heading', name: 'Not on the page')).to be_visible
```
Looking at the logs, it seems like the library is calling [Locator#isVisible](https://playwright.dev/docs/api/class-locator#locator-is-visible) behind the scenes, which has a warning against using it for assertions.

```
SEND>{:id=>6, :guid=>"frame@b6bec03ca1f67ad6b7a463cf11236f6b", :method=>"isVisible", :params=>{:selector=>"internal:role=heading[name=\"Not on the page\"i]", :strict=>true}}
RECV>{"id"=>6, "result"=>{"value"=>false}}
```

What's the correct way to replicate the original snippet?

**Comment by YusukeIwaki** [comment]
Thank you for your feedback. Locator assertion is not yet implemented.

**Comment by abr-storm** [comment]
Are you accepting contributions in this area? It would be cool to have this in the library

**Comment by YusukeIwaki** [comment]
Hi, Pull Request for this feature is really welcome ❤️, thank you!!

**Comment by YusukeIwaki** [comment]
Assertion API is available on Playwright protocol. What we have to consider is that `expect` is reserved 

The first reference existing impl is Playwright for Python.
https://playwright.dev/python/docs/test-assertions
https://github.com/microsoft/playwright-python/blob/main/playwright/_impl/_assertions.py

**Comment by YusukeIwaki** [comment]
@abr-storm Thank you for your great contribution. The feature is now available on 1.39.1 and 1.38.1. Please try it.

**Comment by abr-storm** [comment]
Nice! I'll add it to our app and try it out, thanks for your work on fixing compatibility / docs
